nmdc_server/config.py

```diff
@@ -35,9 +35,9 @@ class Settings(BaseSettings):
     oauth_token_endpoint: str = "https://orcid.org/oauth/token"
     host: Optional[str] = None  # sets the host name for the oauth2 redirect
 
-    # mongo database used for ingest
-    mongo_host: str = "mongo-loadbalancer.nmdc-runtime-dev.development.svc.spin.nersc.org"
-    mongo_port: int = 27017
+    # mongo database used for ingest (see README for how to SSH tunnel to the mongo DB)
+    mongo_host: str = "host.docker.internal"
```

Comment:
It looks like we aren't setting this via the environment in our dev and prod deployments. Would it be preferable to have the default value here be the deployment value, or keep the default as the docker value and set env vars on the deployments?